samples/apps/copilot-chat-app/webapp/src/components/chat/typing-indicator/TypingIndicatorRenderer.tsx

```diff
@@ -13,26 +13,23 @@ const useClasses = makeStyles({
 });
 
 interface TypingIndicatorRendererProps {
-    isBotTyping: boolean;
+    botResponseStatus: string | undefined;
     numberOfUsersTyping: number;
 }
 
-export const TypingIndicatorRenderer: React.FC<TypingIndicatorRendererProps> = ({ isBotTyping, numberOfUsersTyping }) => {
+export const TypingIndicatorRenderer: React.FC<TypingIndicatorRendererProps> = ({
+    botResponseStatus,
+    numberOfUsersTyping,
+}) => {
     const classes = useClasses();
 
-    let message = '';
-    if (isBotTyping) {
-        if (numberOfUsersTyping === 0) {
-            message = 'Bot is typing';
-        } else if (numberOfUsersTyping === 1) {
-            message = 'Bot and 1 user are typing';
-        } else {
-            message = `Bot and ${numberOfUsersTyping} users are typing`;
-        }
-    } else if (numberOfUsersTyping === 1) {
-        message = '1 user is typing';
+    let message = botResponseStatus;
+    if (numberOfUsersTyping === 1) {
+        message = message ? `${message} and 1 user is typing` : '1 user is typing';
```

Comment:
Nit: instead of saying "1 user", maybe write "a user"?
```suggestion
        message = message ? `${message} and a user is typing` : 'A user is typing';
```